assets/scss/local/origin.scss

```diff
@@ -0,0 +1,41 @@
+@charset "UTF-8";
+
+/**
+ * ”Try to imagine all life as you know it stopping instantaneously and every
+ * molecule in your body exploding at the speed of light.” — Egon Spengler
+ *
+ * That’s what’ll happen if you a) modify any global files, or b) change the
+ * order in which any of the files below are imported.
+ *
+ * Note: unfortunately interpolation in Sass isn’t yet available within the
+ * scope of `@import`. Until it is (or we come up with a workaround) we have to
+ * hard-code the paths to each global file.
+ */
+
+/* Local: font settings */
+@import "fonts/_font-settings";
+
+/* Global: functions to handle fonts correctly based on font settings */
+@import "../../../node_modules/origin-css/assets/scss/global/fonts/_font-functions";
+
+/* Global: load fonts for `@font-face` use */
+@import "../../../node_modules/origin-css/assets/scss/global/fonts/_source-sans-pro";
```

Comment:
Imported partial `../../../node_modules/origin-css/assets/scss/global/fonts/_source-sans-pro` should be written as `../../../node_modules/origin-css/assets/scss/global/fonts/source-sans-pro`